doc/Language/typesystem.pod6

```diff
@@ -258,6 +258,49 @@ you must use private attribute notation instead.
     C.new.say; C.new('answer').say;
     # OUTPUT: «C.new(attr => 42)␤
     #          C.new(attr => "answer")␤»
+    
+=head4 submethod TWEAK
+
+The L<submethod|/type/Submethod> C<TWEAK> is (indirectly) called by
+L<.bless|/type/Mu#method_bless>. It
```

Comment:
It's indirectly called when an object is created, even if it does not call explicitly `bless`

Reply:
I used EXACTLY the same words as for submethod BUILD. If the explanation for TWEAK is wrong, then so is the explanation for BUILD. 

My understanding is exactly as documented. 'new' calls 'bless' which calls all BUILDs and then all TWEAKs.

Reply:
Yeah... maybe it's not the best explanation either... My point anyway was that there was no need in the explanation to involve `bless`. Quite enough if you just say when it happens.

Reply:
bless is mentioned several times in the documentation with respect to BUILD, see for example the page on Classes. 

I am not enough of an expert to say whether an object can only be instantiated using bless. But I do not see how to get rid of bless in my explanation of TWEAK if it exists in the explanation of BUILD. They are closely linked submethods.

Reply:
Again, I'm not saying that this is incorrect, just that it's irrelevant. If it's mentioned related to BUILD, that's quite enough, we don't need to go all over again there. TWEAK happens after BUILD, which obviously means it's happened when called from bless. Been there, done that. No need to mention it all over again.

Reply:
This seems resolved